include/drivers/shiftx_drv.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Race Capture Firmware
+ *
+ * Copyright (C) 2016 Autosport Labs
+ *
+ * This file is part of the Race Capture firmware suite
+ *
+ * This is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details. You should
+ * have received a copy of the GNU General Public License along with
+ * this code. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _SHIFTX_DRV_H_
+#define _SHIFTX_DRV_H_
+
+#include "CAN.h"
+
+CPP_GUARD_BEGIN
+
+struct shiftx_configuration {
+        uint8_t orientation_inverted;
```

Comment:
Alignment of these fields is sub optimal.  move the Uint32_t fields up and group the 3 uint8_t types together for better compaction

Reply:
done. Good catch.